lib/actions/build.coffee

```diff
@@ -0,0 +1,81 @@
+# Imported here because it's needed for the setup
+# of this action
+dockerUtils = require('../utils/docker')
+
+exports.getBundleInfo = getBundleInfo = (options) ->
+	Promise = require('bluebird')
+	helpers = require('../utils/helpers')
+
+	if options.application?
+		# An application was provided
+		Promise.try(-> helpers.getAppInfo(options.application))
```

Comment:
Do we need this `Promise.try`? Doesn't `helpers.getAppInfo` return a promise already?